src/main/java/org/mifos/processor/bulk/zeebe/worker/AccountLookupWorker.java

```diff
@@ -68,14 +71,16 @@ public void setup() {
             exchange.setProperty(HOST, identityMapperURL);
             exchange.setProperty(PAYEE_IDENTITY, payeeIdentity);
             exchange.setProperty(PAYMENT_MODALITY, paymentModality);
-            exchange.setProperty(CALLBACK, bulkURL + "/accountLookup/Callback");
+            exchange.setProperty(CALLBACK, identityMapperURL + "/accountLookup/callback");
```

Comment:
these API endpoints should be coming from the configuration.

Reply:
added the endpoints in yaml file